java/code/src/com/suse/manager/webui/websocket/VirtNotifications.java

```diff
@@ -133,7 +134,14 @@ public void onMessage(Session session, String messageBody) {
                 serverIds.add(newId);
 
                 // Send out a list of the latest pending actions to display
-                User user = (User)session.getUserProperties().get("currentUser");
+                User user = Optional.ofNullable(session.getUserProperties().get("webUserID"))
```

Comment:
I don't like having a variable that is `null` here. Can we assign the `Optional` here and move the `orElse(null)` to the function call that expects `null`?

Reply:
Thanks @lucidd. I just change it.